packages/slice-machine/lib/models/ui/Slice/index.tsx

```diff
@@ -97,7 +97,11 @@ const SliceDescription = ({
     }}
   >
     <Flex>
-      <TextWithTooltip text={slice.model.name} as="h6" />
+      <TextWithTooltip
+        text={slice.model.name}
+        as="h6"
+        sx={{ fontWeight: "600 !important" }}
```

Comment:
bypassing the theme 😂, always makes me laugh

Reply:
i'm sure you had your reasons just funny to see ;)

Reply:
It's because the theme uses the default HTML styling of the `as` tag (in this case h6) 😅